packages/@aws-cdk/aws-eks/test/cluster.test.ts

```diff
@@ -2381,12 +2414,12 @@ describe('cluster', () => {
         endpointAccess: eks.EndpointAccess.PUBLIC,
       });
 
-      const nested = stack.node.tryFindChild('@aws-cdk/aws-eks.KubectlProvider') as cdk.NestedStack;
-      const template = SynthUtils.toCloudFormation(nested);
-
-      // we don't attach vpc config in case endpoint is public only, regardless of whether
-      // the vpc has private subnets or not.
-      expect(template.Resources.Handler886CB40B.Properties.VpcConfig).toEqual(undefined);
+      // const nested = stack.node.tryFindChild('@aws-cdk/aws-eks.KubectlProvider') as cdk.NestedStack;
```

Comment:
Is this commented out test (and the ones below) still a WIP?

Reply:
Thanks for spotting this! Mostly it was dead commented out code. But there was one test that I left behind.